frontend/app/views/top_containers/bulk_operations/_bulk_action_templates.html.erb

```diff
@@ -227,7 +227,45 @@
     </div>
 
     <div class="modal-footer">
-        <button type="submit" class="btn btn-danger merge-button" disabled="disabled"><%= I18n.t("top_container._frontend.bulk_operations.merge_n_records", {:n => "${selection.length}"}) %></button>
+        <button class="btn btn-danger merge-button" disabled="disabled" data-js="merge"><%= I18n.t("top_container._frontend.bulk_operations.merge_n_records", {:n => "${selection.length}"}) %></button>
+        <button class="btn btn-cancel" data-dismiss="modal"><%= I18n.t("actions.cancel") %></button>
+    </div>
+
+  <% end %>
+
+--></div>
+
+<div id="bulk_action_merge_confirm"><!--
+  <%= form_tag({:controller => :top_containers, :action => :batch_merge}, {:class => "form-horizontal", :id => "batch_merge_form" }) do |f| %>
+
+    <div class="modal-body">
+
+      <p class="alert alert-danger" id="alertBucket" role="alert">Are you sure you want to merge the top containers as listed below?</p>
```

Comment:
Ok, now that we know the language is staying essentially as is, another small task to put on the roadmap is to rework the hard-coded English-language instructions on lines 243, 245, and and 251 into translations along the lines of what you see in line 208.  Then the English-language phrasing can be moved to: frontend/config/locales/en.yml.  In that same locales directory there are also internationalization files for Spanish, French, and Japanese.  Keys/values should be added there for your 3 new values to keep tests passing.  Your call whether you want to provide translations for the values (Laney and I both blindly use Google Translate and hope users will alert us to any super bad whoopsies), but at the least the keys and English values should be there to keep the files consistent.

Thanks!

Reply:
Completed this using google translate as mentioned, via 8a5fee0.